anesthetic/utils.py

```diff
@@ -243,12 +253,22 @@ def iso_probability_contours(pdf, contours=[0.68, 0.95]):
     return c
 
 
-def iso_probability_contours_from_samples(pdf, contours=[0.68, 0.95],
+def iso_probability_contours_from_samples(pdf, contours=[0.95, 0.68],
                                           weights=None):
     """Compute the iso-probability contour values."""
+    if not np.all(contours[:-1] > contours[1:]):
```

Comment:
This check will throw the error if `contours` is a length 1 array. Sadly I think this is actually a legitimate use case, so we probably want this to be a `>=`. This should also be added to the test suite.

Reply:
For just one element there is no need to test for the order, right... so I'll just modify the conditional to exclude length 1 arrays.